Purchases/Public/CustomerInfo.swift

```diff
@@ -216,6 +216,21 @@ import Foundation
         self.allPurchases = subscriberData.allPurchases
     }
 
+    static func customerInfo(fromJSON maybeJSON: [String: Any]?) throws -> CustomerInfo {
```

Comment:
I would change `CustomerInfo.customerInfo(fromJSON:)` to `CustomerInfo.from(json: )` to avoid the repetition.

Reply:
✅

Reply:
I really like this, ✅